lib/RNCheckboxCard.tsx

```diff
@@ -150,16 +151,18 @@ export default class RNCheckboxCard extends React.Component<
       rightIconComponent,
       sortIconImageSource = defaultSortIcon,
       ImageComponent = Image,
+      disableSortIcon = false,
     } = this.props;
     return (
       <View style={styles.rightIconContainer}>
-        {rightIconComponent || (
-          <ImageComponent
-            resizeMode="contain"
-            source={sortIconImageSource}
-            style={styles.sortIconImageStyle}
-          />
-        )}
+        {rightIconComponent ||
+          (!disableSortIcon && (
```

Comment:
Why do we need this? we can use `rightIconComponent={null}` instead of another prop?

Reply:
If I set this prop to null, six dots icon is shown but I don’t want that

Reply:
Yep you're right, because I am checking with "OR". So we can use 
```
rightIconComponent={<View />}
```

I still don't believe we need this prop. If you really believe that, we can add this prop. What do you think @sevvaleygul0

Reply:
Yes this works, thanks 🙏🏻